android/festago/app/src/main/java/com/festago/festago/presentation/ui/ticketreserve/viewHolder/TicketReserveViewHolder.kt

```diff
@@ -17,10 +17,14 @@ class TicketReserveViewHolder(
         binding.vm = vm
     }
 
-    fun bind(item: ReservationStageUiModel) {
+    fun bind(item: TicketReserveItemUiModel) {
         binding.stage = item
 
-        if (item.canReserve) {
+        if (!item.isSigned) {
```

Comment:
```suggestion
        when {
            !item.isSigned -> {
                binding.btnReserveTicket.isEnabled = true
                binding.btnReserveTicket.text =
                    binding.root.context.getString(R.string.ticket_reserve_tv_signin)
            }

            item.canReserve -> {
                binding.btnReserveTicket.isEnabled = true
                binding.btnReserveTicket.text =
                    binding.root.context.getString(R.string.ticket_reserve_tv_btn_reserve_ticket)
            }

            else -> {
                binding.btnReserveTicket.isEnabled = false
                val pattern = DateTimeFormatter.ofPattern(
                    binding.root.context.getString(R.string.ticket_reserve_tv_btn_reserve_ticket_not_open),
                )
                binding.btnReserveTicket.text = item.ticketOpenTime.format(pattern)
            }
        }
```

3개 이상은 when이 [공식문서](https://developer.android.com/codelabs/basic-android-kotlin-compose-conditionals#2) 권장되고 있습니다!

Reply:
반영했습니다~